### ISSUE TITLE
Lag After Act III Update

### ISSUE THREAD

**Issue by faruqii** [ValveSoftware/Dota2-Gameplay#19983]
### Description

so after the updates, many players have the same issues, the gameplay is laggy, and the response time of movement is so slow, although the connection is stable. Please fix this issue immediately!

**Comment by Madsx96** [comment]
That is completely true. I also have this problem specially at the beginning of the matches.

**Comment by judedaryl** [comment]
im having it right now D: i thought it was me but when i checked my ping i had 16ms ping

**Comment by judedaryl** [comment]
fog of war and normal vision are also inverted, sometimes its normal vision but when i move my mouse i get fog of war shading ( i can see units and enemies but the background is dark )

**Comment by BratovChannel** [comment]
Yes, I have the same

**Comment by faruqii** [comment]
> > fog of war and normal vision are also inverted, sometimes its normal vision but when i move my mouse i get fog of war shading ( i can see units and enemies but the background is dark ) 
> 
> found out my other bug was a macOS sonoma issue
> 
> but i also have this bug where the game lags even though my ping is fine. all actions are delayed including chat, mouse clicks, keyboard press

hope valve fix it soon

**Comment by ryxxed** [comment]
Also in addition to what the issue stated, every time I have used ALT TAB, this problem also appears as well. Nothing to do with ping or packet loss from what I have tested. It goes away after ~30-60s.

**Comment by VetaLioSTV** [comment]
I have the same, please fix asap!

**Comment by IgorHoruzha** [comment]
Do anybody still have the issue?
